package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "parse-server-migrating-adapter",
+  "version": "0.1.0",
+  "engines": {
+    "node": ">=4.0"
+  },
+  "description": "Parse Server file adapter to migrate between adapters",
+  "author": "jemonjam <jacob.e.meacham@gmail.com>",
+  "license": "MIT",
```

Comment:
good question! Palantir seems to be using Apache. For something as simple as this, I think MIT is fine but it's worth looking into as we start open-sourcing more interesting code